org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/manager/wizards/TableButtonControls.java

```diff
@@ -0,0 +1,73 @@
+package org.openjdk.jmc.console.ext.agent.manager.wizards;
+
+import org.eclipse.jface.viewers.TableViewer;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+
+public class TableButtonControls extends Composite {
+	private static final String ADD_BTN_LABEL = "Add";
+	private static final String EDIT_BTN_LABEL = "Edit";
+	private static final String REMOVE_BTN_LABEL = "Remove";
```

Comment:
Perhaps we should add a duplicate button here like how it is in the preset wizard? And perhaps make this a general buttons class that can be use in most of the wizards?

Reply:
`TableInspector` is refactored to handle buttons as well.